scm/driver/bitbucket/git.go

```diff
@@ -28,13 +29,25 @@ func (s *gitService) CreateBranch(ctx context.Context, repo string, params *scm.
 }
 
 func (s *gitService) FindBranch(ctx context.Context, repo, name string) (*scm.Reference, *scm.Response, error) {
+	if strings.Contains(name, "/") {
+		path := fmt.Sprintf("/2.0/repositories/%s/refs/branches?q=name=\"%s\"", repo, name)
+		out := new(branches)
+		res, err := s.client.do(ctx, "GET", path, nil, out)
+		return convertBranch(out.Values[0]), res, err
+	}
 	path := fmt.Sprintf("2.0/repositories/%s/refs/branches/%s", repo, name)
 	out := new(branch)
 	res, err := s.client.do(ctx, "GET", path, nil, out)
 	return convertBranch(out), res, err
 }
 
 func (s *gitService) FindCommit(ctx context.Context, repo, ref string) (*scm.Commit, *scm.Response, error) {
+	if strings.Contains(ref, "/") {
```

Comment:
`FindCommit` is meant to accept a commit sha as opposed to a branch, so I'm not sure we need to make this change. If one is trying to lookup the commit for a branch, they should be using `FindBranch`